app/store/reducers/onboardingReducer.js

```diff
@@ -0,0 +1,19 @@
+import { ONBOARDING_COMPLETE } from '../actions/constants';
+
+const initialState = {
+  complete: false,
+};
+
+const onboardingReducer = (state = initialState, action) => {
```

Comment:
```suggestion
const onboardingReducer = (state = initialState, action) => {
  Object.freeze(state)
```

Reply:
I think we can turn on some dev mode only enhancers that enforce for immutable states. Rather than doing it manually in every reducer.. thoughts?

Reply:
redux toolkit includes some middleware that checks immutable state:

https://redux-toolkit.js.org/api/getDefaultMiddleware#included-default-middleware